modelskill/utils.py

```diff
@@ -17,6 +17,8 @@
     "north": "y",
     "x": "x",
     "y": "y",
+    "z": "z",
+    "depth": "z",
```

Comment:
You're right. But I don't know how common it would be to have both x and longitude as coords? depth and z at the same time could make sense. I think?